openedx/core/lib/xblock_builtin/xblock_video/xblock_video/video_xfields.py

```diff
@@ -4,6 +4,8 @@
 import datetime
 
 from xblock.fields import Scope, String, Float, Boolean, List, Dict, DateTime
+
+# TODO JV: Move to XBlock if only used by Video?
 from xmodule.fields import RelativeTime
```

Comment:
Note to self: Can move this field into `XBlock.fields` as part of https://github.com/edx/XBlock/pull/419 too, since there doesn't appear to be anything else in edx-platform using that class, I can also move that field type into `XBlock.fields`. 

Will need to migrate the [RelativeTime JS editor](https://github.com/edx/edx-platform/blob/a439d5164c07e4695181b15244e8e5c7681421c4/cms/static/js/views/metadata.js#L364) and [tests](https://github.com/edx/edx-platform/blob/a439d5164c07e4695181b15244e8e5c7681421c4/cms/static/js/spec/views/metadata_edit_spec.js#L518) to [use the editor logic in xblock-utils](https://github.com/edx/xblock-utils/blob/18fcd70ff8eed6af17d41b16cf66a8161c9fb51b/xblockutils/studio_editable.py#L96-L108).